CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ## 0.6.1 (???)
```

Comment:
Since this is a new feature, we probably should bump the version to 0.7

Reply:
True